packages/powershell/oh-my-posh/oh-my-posh.psm1

```diff
@@ -20,7 +20,7 @@ function Get-PoshCommand {
 }
 
 # Set the right binary to executable before doing anything else
-if (!$IsWindows) {
+if (!$IsWindows  -and (Get-Item "Variable:*IsWindows*")) {
```

Comment:
Can we change this to: `if ($PSVersionTable.PSEdition -eq "Core" -and !$IsWindows)`
Which means we can change `oh-my-posh.psm1` to include 5 as well within the scope of this PR and use `feat: support Powershell 5` as a commit message.

Reply:
How can i add only last commit to this pull request? Or you can close this request and add it manually

Reply:
If you need to know how, learning to rebase is key here -> https://www.algolia.com/blog/master-git-rebase/
I can always pick this change, you stay the author in that case, let me know what would work for you.

Reply:
Just pick the change that will be good.